### ISSUE TITLE
[Question] What hardware is needed

### ISSUE THREAD

**Issue by khashayar** [henry-spanka/homebridge-buschjaeger#11]
I have the [Indoor Video Station](https://www.busch-jaeger-katalog.de/artikel.php?bereich=1016301&programm=1016445&gruppe=1064673&produkt=1062042) installed and I was wondering, whether any other hardware is also needed in order to be able to communicate with the indoor video station, like [Busch-Welcome IP-Gateway](https://www.busch-jaeger.de/produkte/systeme/tuerkommunikation/busch-welcome-ip-gateway/)

Thank you.

**Comment by henry-spanka** [comment]
Hi,
I think what you're asking is what you need to access the video stream via HomeKit right?

I thought about that too and indeed the only reasonable option is to go with the Busch-Welcome IP-Gateway. However the Camera resolution is bad and you need to reverse engineer the protocol with which the phone app communicates with the IP-Gateway. Also an IP-Gateway is quite expensive. What I did is to buy an IP Camera instead which I installed in the drive way. The advantage is that you can see the whole driveway and the resolution is much better. It's easily integrated in HomeKit and I even have Motion Detection setup, so If the DoorBell rings or there is movement in the drive way when nobody is home, I get a notification with a picture on my iPhone.

So to sum this up:
IP-Gateway is not supported as I don't have one. You need to reverse engineer the communication with a device like a phone to be able to view the video stream.

Let me know if you have any more questions. I have a IP camera with IR for 120 EUR which is far cheaper than an IP-Gateway. And you have more functionality.

**Comment by henry-spanka** [comment]
As I haven't received any reply, I believe the issue is resolved. I'll close the issue for now. Let me know if I should reopen it.
